app/libs/__mocks__/getUser.js

```diff
@@ -5,7 +5,7 @@ const include = [
     model: db.models.userGroup,
     as: 'groups',
     attributes: {
-      exclude: ['id', 'owner_id', 'deletedAt', 'updatedAt', 'createdAt', 'updatedBy'],
```

Comment:
does this have to be removed in the actual getUser.js too in addition to the mock?

Reply:
yes! thank you for pointing it out